configure.ac

```diff
@@ -2042,12 +2042,19 @@ AC_SUBST(PICFLAG)
 # build of the Rust frontend in case a dep is missing.
 missing_rust_dynlibs=none
 
-save_LIBS="$LIBS"
-LIBS=
-AC_SEARCH_LIBS([dlopen], [dl])
-AC_SEARCH_LIBS([pthread_create], [pthread])
-CRAB1_LIBS="$LIBS"
-LIBS="$save_LIBS"
+case "${target}" in
+  *-*-mingw*)
+      CRAB1_LIBS="-lkernel32"
+	;;
+  *)
+    save_LIBS="$LIBS"
+    LIBS=
+    AC_SEARCH_LIBS([dlopen], [dl])
+    AC_SEARCH_LIBS([pthread_create], [pthread])
+    CRAB1_LIBS="$LIBS"
+    LIBS="$save_LIBS"
+  ;;
```

Comment:
The `;;` indentation is inconsistent with the block above.

Reply:
I wasn't too sure how to indent that. Should the semicolon be aligned with the case ?

```
case "${target}" in
LIBS=
  *-*-mingw*)
AC_SEARCH_LIBS([dlopen], [dl])
      CRAB1_LIBS="-lkernel32 -luser32 -lmsvcrt -lmingwex -lmingw32 -lmingwex"
AC_SEARCH_LIBS([pthread_create], [pthread])
;;
CRAB1_LIBS="$LIBS"
  *)
LIBS="$save_LIBS"
    save_LIBS="$LIBS"
    LIBS=
    AC_SEARCH_LIBS([dlopen], [dl])
    AC_SEARCH_LIBS([pthread_create], [pthread])
    CRAB1_LIBS="$LIBS"
    LIBS="$save_LIBS"
;;
```

Reply:
Honestly, either style is fine (people do a mix) but just have it be consistent. I personally have it aligned with the last statement inside the case usually (so the 1st one).

GCC seems to usually do that too (first variant).